core/src/main/scala/org/http4s/headers/Sec-WebSocket-Version.scala

```diff
@@ -17,4 +17,33 @@
 package org.http4s
 package headers
 
-object `Sec-WebSocket-Version` extends HeaderKey.Default
+import org.http4s.parser.HttpHeaderParser
+import org.http4s.util.{Renderer, Writer}
+import scala.concurrent.duration.{FiniteDuration, _}
+import scala.util.Try
+
+object `Sec-WebSocket-Version`
```

Comment:
am i implementing all these headers w/ the new model correctly?

Reply:
The new model is in 0.22 and up, so this looks fine for 0.21.
I do think we should eliminate backticks for headers, but that is a different discussion.